index.d.ts

```diff
@@ -52,7 +52,7 @@ declare module 'webpack-plugin-serve' {
   interface Builtins {
     proxy: (args: HttpProxyMiddlewareConfig) => Proxy;
     compress: (opts: CompressOptions) => void;
-    static: (opts: KoaStaticOptions) => void;
+    statik: (paths: Array<String>, opts?: KoaStaticOptions) => void;
```

Comment:
@PlayMa256 going to revert this change. it's `static` and not `statik`. see: https://github.com/shellscape/webpack-plugin-serve/pull/83/files#diff-d7dfcb350720309421bbb465055aafdfR126

Reply:
missread that. My bad, reverted.